src/lib/api.js

```diff
@@ -94,25 +94,23 @@ const api = {
    */
   available: async () => {
     const url = `${environment.apiUrl}/`;
+    log.info(`Checking that ${url} is available...`);
     try {
-      log.info(`Checking that ${url} is available...`);
       await request.get(url);
-      return true;
     } catch (err) {
       if (err.statusCode === 401) {
-        log.error(`Authentication failed connecting to ${url}. `
+        throw new Error(`Authentication failed connecting to ${url}. `
                   + 'Check the supplied username and password and try again.');
       } else if (err.statusCode === 403) {
```

Comment:
We don't need all these `else`s here anymore, right?

Reply:
True. Fixed.